data/battle_scripts_1.s

```diff
@@ -7058,6 +7058,7 @@ BattleScript_ObliviousPreventsAttraction::
 
 BattleScript_FlinchPrevention::
```

Comment:
Since Inner Focus doesn't have a pop-up or text when it blocks certain Flinch effects (for context, it only says Inner Focus activated when a move has Flinching as its primary effect or if the flinch effect is certain/has 100% chance like with Fake Out), this battle script can be removed from here and `include/battle_scripts.h`.